packages/store-api/src/typeDefs/aggregateOffer.graphql

```diff
@@ -0,0 +1,7 @@
+type StoreAggregateOffer {
```

Comment:
I'm in doubt if we should put that all fields are required, because maybe this is a VTEX specificity, it could be that other platforms only have the price. Shouldn't I leave some of these fields as optional?

Reply:
I'm not sure. I think it should be required because, in the end of the day, we need to render these properties in our fronted anyways. If the store doesn't use an specific property it can just not query it, right?